### DISCUSSION TITLE
Use of Zarr and other data format issues

### DISCUSSION THREAD

**Discussion by zklaus** [ESMValGroup/ESMValCore#1621] [General]
Lately, there has been a lot of talk about new or alternative data formats that might be better suited for the cloud.

On this topic, there is now [support for zarr in netcdf](https://www.unidata.ucar.edu/blogs/developer/en/entry/overview-of-zarr-support-in) and there is a mapping to zarr in xarray.

It might also be worthwhile discussing how much of the goals can be achieved with good use of features in netcdf, as well as what others are doing, see e.g. [this discussion in pangeo](https://discourse.pangeo.io/t/netcdf-to-zarr-best-practices/1119).

**Comment by bouweandela** [comment]
More information on current efforts can be found here: https://pangeo-data.github.io/pangeo-cmip6-cloud/
